modules/alerting/main.tf

```
@@ -199,8 +199,8 @@ resource "google_monitoring_alert_policy" "signal" {
 locals {
   panic_filter = <<EOF
 resource.type="cloud_run_revision" OR resource.type="cloud_run_job"
-severity=ERROR
-textPayload=~"panic: .*"
+severity<=ERROR
+textPayload=~"panic: .*|runtime/panic"
```

```suggestion
textPayload=~"panic: .*" OR textPayload=~"runtime/panic"
```

Small thing, but I never trust regexes when there's a real boolean operator available.